src/settings/_measurements.scss

```diff
@@ -15,6 +15,20 @@
 
 $govuk-page-width: 960px !default;
 
+/// Map of grid column widths
+///
+/// @type Map
+/// @access public
+
+$govuk-grid-widths: (
+  one-quarter: 25%,
+  one-third: 33.3333%,
```

Comment:
Quick question. Is it worth letting the Sass language handle the [precision here](http://sass-lang.com/documentation/Sass/Script/Value/Number.html#precision-class_method)?

Swapping this out for:

```
one-third: (100/3) * 1%,
```

I believe this is possible in a map as according to the docs: "Both the keys and values in maps can be any SassScript object".

I doubt it would have any effect on the rounded result so may just be best left hardcoded.

Reply:
Might be worth keeping it simple since it works and doesnt require thinking.